src/browserlib/extract-pages.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Extract pages from the table of contents of the current document.
+ *
```

Comment:
sounds like out of scope of this pull request :)

Reply:
Indeed :) Dropped.